agate/agate/queue_reading/ingestion_updater.py

```diff
@@ -0,0 +1,79 @@
+import logging
+from agate.models import IngestionAttempt
+from agate.forms import IngestionAttemptForm
+
+logger = logging.getLogger(__name__)
+
+
+class IngestionUpdater:
+    """
+    This class interprets a dictionary (as may come from a rabbitMQ message)
+    and creates or updates an IngestionAttempt appropriately.
+    """
+
+    @classmethod
+    def update(cls, data: dict, stage: str):
+        """
+        Use a dictionary to update items in the database
+        """
+        try:
+            uuid = data["uuid"]
+        except KeyError:
+            logger.fatal("no uuid")
```

Comment:
This is the first time I've seend the `FATAL` level used, which took me down a rabbit hole to where Python's logging system was specified ([PEP 282](https://peps.python.org/pep-0282/)). The PEP advises:

> The term CRITICAL is used in preference to FATAL, which is used by log4j. The levels are conceptually the same - that of a serious, or very serious, error. However, FATAL implies death, which in Python implies a raised and uncaught exception, traceback, and exit. Since the logging module does not enforce such an outcome from a FATAL-level log entry, it makes sense to use CRITICAL in preference to FATAL.

So I'd suggest
```suggestion
            logger.critical("no uuid")
```
wherever you current use `logging.fatal`.